.gitattributes

```diff
@@ -19,6 +19,7 @@ tests/Core/**/                      export-ignore
 phpcs.xml.dist                      export-ignore
 phpstan.neon.dist                   export-ignore
 phpunit.xml.dist                    export-ignore
+phpunitlt10.xml.dist                export-ignore
```

Comment:
The first thing that stood out to me when reading the file name is why I'm seeing 10 if this file is for PHPUnit 9 and 8? Only after a while I noticed the `lt` before 10. I suggest using `lte9` instead of `lt10`. I think using hyphens also helps with the readability of the filename, so `phpunit-lte-9.xml.dist`. What do you think?

Reply:
Good point, I've updated the PR to action this.